settings.py

```diff
@@ -0,0 +1,10 @@
+import os
+from dotenv import load_dotenv
+
+load_dotenv()
+
+# AWS configuration credentials
+AWS_CONFIG = {
+    "aws_access_key_id": os.getenv("ACCESS_KEY"),
```

Comment:
Let's be specific with the environment variable names or it might conflict with other stuff.

```py
AWS_CONFIG = {
  "aws_access_key_id": os.getenv("AWS_ACCESS_KEY_ID"),
  "aws_secret_access_key": os.getenv("AWS_SECRET_ACCESS_KEY")
}
```